Dockerfile

```diff
@@ -1,13 +1,31 @@
-FROM golang:1.13.0 AS build
+FROM golang:1.13 AS build
```

Comment:
We would like to have control over the patch version of Go as well.
Not pinning the patch version can result in using an older version of Go due to Docker's cache.

Please revert this change.